supervisor/stats.go

```diff
@@ -17,24 +16,82 @@ func (h *StatsEvent) Handle(e *Event) error {
 	if !ok {
 		return ErrContainerNotFound
 	}
-	e.Stats = h.s.statsCollector.collect(i.container)
+	st, err := i.container.Stats()
+	if err != nil {
+		return err
+	}
+	e.Stats = convertToPb(st)
```

Comment:
Should these conversions happen in the API package cuz this brings in the types import into the supervisor package?